nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/java/org/apache/nifi/processors/gcp/bigquery/BigQueryAttributes.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.nifi.processors.gcp.bigquery;
+
+import org.apache.nifi.components.PropertyDescriptor;
+import org.apache.nifi.processors.gcp.credentials.factory.CredentialPropertyDescriptors;
+
+/**
+ * Attributes associated with the BigQuery processors
+ */
+public class BigQueryAttributes {
+    private BigQueryAttributes() {}
+
+    public static final PropertyDescriptor SERVICE_ACCOUNT_JSON_FILE = CredentialPropertyDescriptors.SERVICE_ACCOUNT_JSON_FILE;
+
+    public static final String DATASET_ATTR = "bq.dataset";
+    public static final String DATASET_DESC = "BigQuery dataset";
+
+    public static final String TABLE_NAME_ATTR = "bq.table.name";
+    public static final String TABLE_NAME_DESC = "BigQuery table name";
+
+    public static final String TABLE_SCHEMA_ATTR = "bq.table.schema";
+    public static final String TABLE_SCHEMA_DESC = "BigQuery table name";
```

Comment:
copy/paste error.
besides the description should specify that the schema must be provided as JSON data. At first I tried to specify the schema with ``name:STRING,age:INTEGER,email:STRING`` which is also an option in the BigQuery UI.